ansible/group_vars/all.yml

```diff
@@ -31,8 +31,8 @@ stanchion_data_dir: "{{ encrypted_root }}/stanchion"
 # commcare-hq connects to an S3-compatible service, which is Riak CS
 s3_blob_db_enabled: "{{ 'true' if groups.get('riakcs') else '' }}"
 s3_blob_db_url: "http://{{ groups['proxy'][0] }}:{{ riakcs_port }}"
-s3_blob_db_access_key: "{{ hostvars[riakcs_control]['riak_key'] if s3_blob_db_enabled else '' }}"
-s3_blob_db_secret_key: "{{ hostvars[riakcs_control]['riak_secret'] if s3_blob_db_enabled else '' }}"
+s3_blob_db_access_key: "{{ hostvars[riakcs_control].get('riak_key', 'not-set') if s3_blob_db_enabled else '' }}"
+s3_blob_db_secret_key: "{{ hostvars[riakcs_control].get('riak_secret', 'not-set') if s3_blob_db_enabled else '' }}"
```

Comment:
> The command that failed was step 8 (deploy the full stack)...

Did it only fail with `--check` (that isn't too surprising, but I should try to fix it), or did it fail without `--check` as well?